src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const tableHead = document.querySelector('thead');
+const cloneTableHead = tableHead.cloneNode(true);
+const tableFoot = document.querySelector('tfoot');
+const cloneTableFoot = tableFoot.cloneNode(true);
+const headPosition = cloneTableHead.querySelector('tr :nth-child(2)');
+const footPosition = cloneTableFoot.querySelector('tr :nth-child(2)');
+const headSalary = tableHead.querySelector('tr :nth-child(5)');
+const footSalary = tableFoot.querySelector('tr :nth-child(5)');
+const tableBody = document.querySelectorAll('tbody tr');
+
+headSalary.before(headPosition);
+footSalary.before(footPosition);
```

Comment:
You are moving the second cell of the header and footer rows before the fifth cell. However, you don't remove the original second cell. This will result in duplicate cells in your header and footer rows. You should remove the original second cell after it has been moved.